CONTRIBUTING.md

```diff
@@ -17,38 +13,28 @@ You can also suggest anything else that you think improves the community for us
 
 The Kedro team pledges to foster and maintain a friendly community. We enforce a [Code of Conduct](./CODE_OF_CONDUCT.md) to ensure every Kedroid is welcomed and treated with respect.
 
-## Join the community and help it grow!
+## Join the community
 
 You can find the Kedro community on our [Discord server](https://discord.gg/akJDeVaxnB), which is where we share news and announcements, and general chat. You're also welcome to post links here to any articles or videos about Kedro that you create, or find, such as how-tos, showcases, demos, blog posts or tutorials.
 
-We occasionally post on the [QuantumBlack blog](https://medium.com/quantumblack/) and we curate a [Github repo that lists content created by the Kedro community](https://github.com/kedro-org/kedro-community).
+We also curate a [Github repo that lists content created by the Kedro community](https://github.com/quantumblacklabs/kedro-community).
 
-## Community Q&A
+## Contribute to the project
 
-We encourage you to ask and answer technical questions on [GitHub discussions](https://github.com/kedro-org/kedro/discussions).
+There are quite a few ways to contribute to the project, find inspiration from the table below.
 
-## Report a bug
+|Activity|Description|
+|-|-|
+|Community Q&A|We encourage you to ask and answer technical questions on [GitHub discussions](https://github.com/quantumblacklabs/kedro/discussions) or [Discord](https://discord.gg/akJDeVaxnB), but the former is often preferable since it will be picked up by search engines.|
```

Comment:
```suggestion
|Community Q&A|We encourage you to ask and answer technical questions on [GitHub discussions](https://github.com/kedro-org/kedro/discussions) or [Discord](https://discord.gg/akJDeVaxnB), but the former is often preferable since it will be picked up by search engines.|
```